Automattic/wp-calypso
# Stats: Remove chart legend

#### Proposed Changes

* removing chart legend
* putting the parent/unique CSS class name for the chart component into a constant variable for easier identification
* applying additional padding at the top of the chart
* removing redundant props from parent components that are used by the legend

#### Testing Instructions

* open the `Stats` page and verify that the legend area above the chart is no longer visible
* verify that x-axis labels are properly placed and don't extend beyond the chart area
* verify that the legend was removed from `Ads` tab (you will need Jetpack Security or Full JP license and you'd have to enable WordAds on one of your JN instances that you will inspect in this Calypso test build). [More info about enabling WordAds](https://jetpack.com/support/ads/#activating-wordads).

Before:
![Screenshot 2022-10-06 at 5 07 37 PM](https://user-images.githubusercontent.com/112354940/194211862-a91f286e-b7d3-46e7-a1a5-9cb6123cfa97.png)

After:
![Screenshot 2022-10-07 at 5 41 52 PM](https://user-images.githubusercontent.com/112354940/194469227-f4a80337-e7b2-4686-9948-aa991eb1e80f.png)



#### Pre-merge Checklist






Fixes #68676
